src/EditorFeatures/CSharpTest/RemoveUnusedParametersAndValues/RemoveUnusedParametersTests.cs

```diff
@@ -83,10 +83,10 @@ public async Task Parameter_Unused_NonPrivate_NotApplicable(string typeAccesibil
         [InlineData("internal", "public")]
         [InlineData("internal", "internal")]
         [InlineData("internal", "protected")]
-        public async Task Parameter_Unused_NonPublicMethod(string typeAccesibility, string methodAccessibility)
+        public async Task Parameter_Unused_NonPublicMethod(string typeAccessibility, string methodAccessibility)
         {
             await TestDiagnosticsAsync(
-$@"{typeAccesibility} class C
```

Comment:
Thanks for fixing the typos!

Reply:
No problem. I introduced on of my own actually (fixed in d7c0209080b27429195f0f7390bf06eda640341c) 😃